addons/base_import_module/models/ir_module.py

```diff
@@ -35,7 +35,7 @@ def _get_latest_version(self):
     def _import_module(self, module, path, force=False):
         known_mods = self.search([])
         known_mods_names = {m.name: m for m in known_mods}
-        installed_mods = [m.name for m in known_mods if m.state == 'installed']
+        installed_mods = [m.name for m in known_mods if m.state != 'unknown']
 
         terp = load_information_from_description_file(module, mod_path=path)
```

Comment:
This line is reading everything from the manifest except the categories (smth like that).

Categories are handled separately after that when you import a module in base (normal way), shouldn't we "just" copy paste those lines here (and then maybe encapsulate it and call a shared method)?

If you investigated in that regard and it didn't work, any feedback about why not?